lib/uniform/uniform.test.tsx

```diff
@@ -40,20 +58,32 @@ describe('<Uniform />', () => {
     expect(wrapper.getDOMNode().getAttribute('class')).toBe('oc-bar');
   });
 
-  it('Handles no-touchevents', () => {
+  it('Handles props.className', () => {
     uniform.hasMouse = jest.fn().mockReturnValue(false);
+    uniform.getMode = jest.fn().mockReturnValue('light');
 
-    wrapper = mount(<Uniform mode=" ">Foo</Uniform>);
+    wrapper = mount(<Uniform>foo</Uniform>);
 
-    expect(wrapper.getDOMNode().getAttribute('class')).toBe('');
+    expect(wrapper.getDOMNode().getAttribute('class')).toBe('oc-light');
 
-    uniform.hasMouse = jest.fn().mockReturnValue(true);
+    wrapper.setProps({ className: 'bar' });
+    expect(wrapper.getDOMNode().getAttribute('class')).toBe('oc-light bar');
 
-    wrapper.unmount();
-    wrapper = mount(<Uniform mode=" ">Foo</Uniform>);
+    wrapper.setProps({ className: 'bar baz' });
+    expect(wrapper.getDOMNode().getAttribute('class')).toBe('oc-light bar baz');
+  });
 
-    expect(wrapper.getDOMNode().getAttribute('class')).toBe(
-      'oc-no-touchevents'
+  it('Handles props.style', () => {
+    wrapper = mount(<Uniform>foo</Uniform>);
+
+    expect(wrapper.getDOMNode().getAttribute('style')).toBe(null);
+
+    wrapper.setProps({ style: { zIndex: '1' } });
```

Comment:
same here... ignore if its useful indeed

Reply:
I'm testing 0, 1 and 2 CSS properties across all the assertions—just to be thorough.